ImageData/HDF5Image.h

```diff
@@ -0,0 +1,58 @@
+//# HDF5Image.h : CARTA HDF5Image derived from casacore::HDF5Image
```

Comment:
Should this not be `casacore::ImageInterface`?

Reply:
Yes, change of plans.  Will fix.